M2/Macaulay2/packages/SimpleDoc/helpers-doc.txt

```diff
@@ -0,0 +1,81 @@
+-- -*- mode: M2; coding: utf-8 -*-
+Node
+  Key
+     wikipedia
+    (wikipedia, String)
+    (wikipedia, String, String)
+  Headline
+    link to a Wikipedia article
+  Usage
+    wikipedia(url, title)
+    wikipedia title
+  Inputs
+    url:String
+      the article's url slug, optional
+    title:String
+      a Wikipedia article's title
+  Outputs
+    :Hypertext
+      a link to the Wikipedia entry with the given title usable inside a documentation node
+  Description
+    Example
+      wikipedia "Bessel function"
+      html oo
+      wikipedia("Bessel_function#Definitions", "Bessel function")
+      html oo
+  SeeAlso
+    document
+    arXiv
+    stacksProject
+Node
+  Key
+     arXiv
+    (arXiv, String)
+    (arXiv, String, String)
+  Headline
+    link to an arXiv article
+  Usage
+    arXiv ref
+    arXiv(ref, title)
+  Inputs
```

Comment:
How would arXiv be used inside a doc string?  Inside at-signs?

Reply:
Yes.

Reply:
I'll mention an example in the documentation for `doc(String)`.